src/NextGen/Gateways/Stripe/NextGenStripeGateway/NextGenStripeRepository.php

```diff
@@ -15,19 +15,16 @@
 use Stripe\Exception\InvalidRequestException;
 use Stripe\PaymentIntent;
 
-trait NextGenStripeRepository {
+trait NextGenStripeRepository
+{
     /**
      * @since 0.1.0
      * @throws ApiErrorException
      */
-    protected function generateStripePaymentIntent($accountId, Money $amount): PaymentIntent
+    protected function generateStripePaymentIntent(string $accountId, array $args): PaymentIntent
```

Comment:
Should the `$args` be typed as a DTO? Otherwise we are assuming that the array contains the correct/valid arguments.

Reply:
Agreed, updated 2be53361